torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -24,6 +24,9 @@
 #include <torch/custom_class.h>
 
 namespace c10d {
+// Environment variable which controls whether we perform a NCCL healt check
```

Comment:
nit: healt -> health (please feel free to update this in followup PRs if this needs to land ASAP)